edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/AuthorizationConfiguration.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Domain object that represents Authorization configuration for Edge Hub Module (MQTT Broker).
+    ///
+    /// This object is being eventually constructed from the EdgeHub twin's desired properties.
+    /// See <see cref="EdgeHubDesiredProperties"/> for DTO.
+    /// </summary>
+    public class AuthorizationConfiguration : List<Statement>, IEquatable<AuthorizationConfiguration>
```

Comment:
Add tests

Reply:
@philipktlin what test cases do you have in mind? i've added tests for deserialization and equality. There is nothing much else to test here for now.